db/schema.rb

```diff
@@ -59,8 +63,8 @@
     t.string   "youtube_id"
     t.string   "title"
     t.text     "description"
-    t.datetime "created_at",   :null => false
-    t.datetime "updated_at",   :null => false
+    t.datetime "created_at",  :null => false
+    t.datetime "updated_at",  :null => false
```

Comment:
For some reason the spacing here was messed up. This is the correct spacing as generated by db:migrate. The previous one was wrong.